lib/pc_patch_lazperf.c

```diff
@@ -70,10 +70,22 @@ pc_patch_lazperf_from_uncompressed(const PCPATCH_UNCOMPRESSED *pa)
 PCPOINTLIST *
 pc_pointlist_from_lazperf(const PCPATCH_LAZPERF *palaz)
 {
-	PCPATCH_UNCOMPRESSED *pu = NULL;
-	pu = pc_patch_uncompressed_from_lazperf(palaz);
-	PCPOINTLIST *pl = pc_pointlist_from_uncompressed(pu);
-	pc_patch_free((PCPATCH *)pu);
+	PCPATCH_UNCOMPRESSED *pu = pc_patch_uncompressed_from_lazperf(palaz);
```

Comment:
That looks good.
Maybe this memory issue fix could be factored out by creating a `pc_pointlist_clone` function to facilitate code reuse and unit testing.